R/main.R

```diff
@@ -11,6 +11,40 @@ check_drjacoby_loaded <- function() {
   message("drjacoby loaded successfully!")
 }
 
+#------------------------------------------------
+#' @title Define parameters dataframe
+#'
+#' @description Provides a convenient way of defining parameters in the format
+#'   required by \code{run_mcmc()}.
+#'
+#' @param ... define parameters one at a time, for each one defining four
+#'   elements: name, min, max, init.
+#'
+#' @export
+

```

Comment:
Could we mabe add an example here - its not clear to me how this is used to define multiple params (can we chain mutiple calls to this function togther somehow?)

Reply:
You're quite right, it was lazy of me not to make an example. Will add one now